apps/ccf-rui/src/app/shared/components/rotation-slider/rotation-slider.component.html

```diff
@@ -1,95 +1,103 @@
 <div class="header">
-  <span class="text title" matTooltip="Use sliders below to rotate tissue block for registration">Rotation</span>
-  <div class="filler"></div>
-  <!-- <div class="refresh-button-container">
-    <mat-icon
-      matRipple
-      [matRippleCentered]="true"
-      [matRippleUnbounded]="true"
-      class="icon reset"
-      (click)="resetRotation('x'); resetRotation('y'); resetRotation('z')"
-      >refresh</mat-icon
-    >
-  </div> -->
+  <span class="text title" hraMicroTooltip="Use sliders below to rotate tissue block for registration">Rotation</span>
+  <span class="text title" hraMicroTooltip="Use sliders below to rotate tissue block for registration">Rotation</span>
 </div>
 
-<div class="slider-box red">
-  <!-- <span class="text label">X</span> -->
-  <!-- <div class="slider-container">
-    <mat-slider class="slider" [max]="180" [min]="-180" disableRipple discrete>
-      <input
-        class="slider-x"
-        matSliderThumb
-        [value]="rotation.x"
-        (input)="changeRotation(sliderX.value, 'x')"
-        #sliderX
-      />
-    </mat-slider>
-  </div> -->
-  <mat-form-field>
-    <mat-label class="text form-input-label">X</mat-label>
-    <input
-      matInput
-      class="input"
-      type="number"
-      [value]="rotation.x"
-      [min]="-180"
-      [max]="180"
-      (change)="changeRotation($any($event.target).value, 'x')"
-    />
-  </mat-form-field>
+<div class="slider-box">
```

Comment:
This slider box should be made into its own component instead of repeating the same big code block multiple times